views/index.erb

```diff
@@ -1,8 +1,31 @@
 <h1><%= @title %></h1>
 
 Output generated: <%= Time.now %>
+
 <pre>
 $ <%= @command %>
-
-<%= @output %>
 </pre>
+
+<table class="table table-bordered">
+  <tr>
+    <th>USER</th>
+    <th>PID</th>
+    <th>%CPU</th>
+    <th>%MEM</th>
+    <th>VSZ</th>
+    <th>RSS</th>
+    <th>TTY</th>
+    <th>STAT</th>
+    <th>START</th>
+    <th>TIME</th>
+    <th>COMMAND</th>
+  </tr>
+  <% @output.each do |row| %>
+  <tr>
+    <% row.each do |value| %>
+      <td><%= value %></td>
+    <% end %>
+  </tr>
```

Comment:
I'm not sure about this. I know its the natural outcome of adding parsing to the command...but this code is a little esoteric:

```ruby
<% @output.each do |row| %>
<tr>
  <% row.each do |value| %>
    <td><%= value %></td>
  <% end %>
</tr>
<% end %>
```

This reads better:

```ruby
<% app_processes.each do |app| %>
<tr>
  <td><%= app.user %></td>
  <td><%= app.pid %></td>
  <%# ... %>
  <td><%= app.time %></td>
  <td><%= app.command %></td>
</tr>
<% end %>
```
We would have a model object that defines the attributes to display, and the parsing method would produce an array of these objects. Maybe a Struct is defined in the config.ru and then used?

```ruby
AppProcess = Struct.new(:user, :pid, :pct_cpu, :pct_mem, :vsz, :rss, :tty, :stat, :start, :time, :command)
```

Of course we are slowly inching our way towards a separate model class in a separate file. Not sure if that is ideal for what we are doing here. Don't want to add too much complexity to confuse. 

Maybe we keep as is. Is there is another way? I don't know.